### ISSUE TITLE
Login

### ISSUE THREAD

**Issue by XeRiZeR** [xMansour/KidSafe#1]
Cant login, just keep saying loading.

**Comment by Kiqe24111980** [comment]
Same Here, stay in "Loading"

**Comment by Manwe-fv** [comment]
Also can't login. Looks like this app is currently broken.

**Comment by jviqueira** [comment]
Yes, after register and verify the account, the main screen stay in "Loading..." screen forever... Android 8.1.0

**Comment by hosios** [comment]
Same thing on my phone. Registerd, verified E-mail, and the Login page just says "Loading..."

Please fix this, we need an app like this. There is no other OpenSource in this category. Family Link is the only free one, but it has its limitations.

Thank you.

**Comment by sidhiqsidhu** [comment]
Can't login the App keep loading....

**Comment by pyscriptor** [comment]
Same problem. Can't login.

**Comment by Paul75** [comment]
Same problem. 
.....

**Comment by 4g4p3** [comment]
January of 2026 and the problem persists...
